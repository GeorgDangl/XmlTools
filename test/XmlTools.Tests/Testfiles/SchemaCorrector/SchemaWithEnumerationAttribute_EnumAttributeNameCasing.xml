<?xml version="1.0" encoding="UTF-8"?>
<Author gender="Male">
  <Email>writer@example.com</Email>
  <Name>John Doe</Name>
</Author>