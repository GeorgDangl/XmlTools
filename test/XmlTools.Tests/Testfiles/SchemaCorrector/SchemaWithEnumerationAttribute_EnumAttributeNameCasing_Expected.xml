<?xml version="1.0" encoding="UTF-8"?>
<Author Gender="Male">
  <Email>writer@example.com</Email>
  <Name>John Doe</Name>
</Author>